api/Navigator.json

```diff
@@ -2513,7 +2513,7 @@
             "firefox_android": {
               "version_added": "79",
               "partial_implementation": true,
-              "notes": "Firefox does not support sharing files."
+              "notes": "Firefox does not support sharing files or text."
```

Comment:
Should it explicitly say it's Android specific issue?

Reply:
Good point. Updated. 👍